core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/StackV4Endpoint.java

```diff
@@ -275,10 +275,10 @@ FlowIdentifier setClusterMaintenanceMode(@PathParam("workspaceId") Long workspac
     FlowIdentifier updateSaltByName(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
     @PUT
-    @Path("{name}/pillar_config_update")
+    @Path("crn/{crn}/pillar_config_update")
```

Comment:
I am not sure that is it affects the UI but probably @keyki know this

Reply:
@horadla23 any authorization is required here ?

Reply:
The whole controller was set to @DisableCheckPermissions by @keyki

Reply:
It should not affect the UI.  I added this in the just previous PR to be called by this explicitly.  It will never have been used by the UI.

Reply:
i have another pr regarding this stack api authorization, for now it is good with disabledcheckpermissions annotation

Reply:
change regarding stack api authorization is merged, please use same right to check permission as used for other stack api methods `@CheckPermissionByAccount(action = AuthorizationResourceAction.USE_INTERNALLY)` and also annotate crn param with `@TenantAwareParam` and it should work

Reply:
added and in